package.json

```diff
@@ -30,7 +30,7 @@
     "semistandard": "^7.0.0",
     "solc": "https://github.com/ethereum/solc-js",
     "tape": "^4.5.1",
-    "web3": "^0.16.0",
+    "web3": "https://github.com/ethereum/web3.js#35bcace7173fda742ddb1bba84c8a70f0fa849e0",
```

Comment:
That hash is the current version of `master`. Asked the web3.js team when a 0.17.0 release is coming, but no response yet.